playbooks/generic-delete_ses_rook.yml

```diff
@@ -15,6 +15,8 @@
 - hosts: soc-deployer
   gather_facts: no
   any_errors_fatal: true
+  vars_files:
+    - "{{ playbook_dir }}/../vars/common-vars.yml"
```

Comment:
is this needed? commit doesnt say why and Im not seeing any vars that are not self contained?

Reply:
It's for the repo path, I will adapt commit message

Reply:
ouch, so this is not standalone anymore then :(